update dump-cov for alignment and writergate changes

dweiller opened the pull request:
I discovered that this tool doesn't compile with the current compiler.

Should I add some CI coverage for Zig programs in `tools/` to ensure they remain working?

alexrp commented:
I guess just add it here: https://github.com/ziglang/zig/blob/d3cd0b27145fb9d295078a6883f9714769cb482b/test/standalone/build.zig#L28-L59

dweiller commented:
I've added `dump-cov` to the standalone tests, and updated `test/standalone/build.zig.zon` for the change to the `.name` field and introduction of the `.fingerprint` field.
